src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -26,17 +34,98 @@ private void initMenu() {
         this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
 
         this.add(this.importMenu);
-        this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
+        this.add(this.settingsMenu);
+
         // importMenu should be disabled  before signing in
-        this.importMenu.setEnabled(false);
+//        this.importMenu.setEnabled(false);
+
+        // Add function to import menu
+        importMenu.addMenuListener(new MenuListener() {
+            @Override
+            public void menuSelected(MenuEvent e) {
+                importBills();
+            }
+
+            @Override
+            public void menuDeselected(MenuEvent e) {
+
+            }
+
+            @Override
+            public void menuCanceled(MenuEvent e) {
+
+            }
+        });
+        this.add(Box.createRigidArea(new Dimension(ActionPanel.VERTICAL_GAP, 0)));
+
+        // Add function to setting menu
+        settingsMenu.addMenuListener(new MenuListener() {
```

Comment:
Since the layout of the pop dialog is different in macOS and Windows, we can make the menu items `JCheckBoxMenuItem` (instead of displaying a dialog). And I think this will solve most of the problems that I mentioned in the previous review. 

For example,
![image](https://user-images.githubusercontent.com/60980117/202801278-613b5480-2a4d-47f7-af00-dda57727620f.png)

However, you could still use your own design, but you have to fix the bugs that I mentioned.